rest/route/user.go

```diff
@@ -245,27 +257,33 @@ func (h *userPermissionsDeleteHandler) Run(ctx context.Context) gimlet.Responder
 		}))
 		return gimlet.MakeJSONInternalErrorResponder(errors.New("unable to get roles for user"))
 	}
-	scopesIds := []string{}
-	for _, role := range roles {
-		scopesIds = append(scopesIds, role.Scope)
+	rolesToCheck := []gimlet.Role{}
+	// don't remove basic access, just special access
+	for _, r := range roles {
+		if !utility.StringSliceContains(evergreen.BasicAccessRoles, r.ID) {
+			rolesToCheck = append(rolesToCheck, r)
+		}
+	}
+	if len(rolesToCheck) == 0 {
+		gimlet.NewJSONResponse(struct{}{})
 	}
-	applicableScopes, err := h.rm.FilterScopesByResourceType(scopesIds, h.resourceType)
+
+	rolesForResource, err := h.rm.FilterForResource(rolesToCheck, h.resourceId, h.resourceType)
 	if err != nil {
-		grip.Error(message.WrapError(err, message.Fields{
```

Comment:
Why are we no longer logging to Splunk here?

Reply:
It's a bit redundant; we already log errors from rest routes.